### ISSUE TITLE
rename group-whitelist and group-blacklist to group-allowlist and group-denylist

### ISSUE THREAD

**Issue by anoonan** [linkedin/Burrow#643]
Language matters. We removing all references to non-inclusive language in our org and would like to be able to update our burrow configs as well.

[Here is an article explaining why language matters](https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white).

**Comment by toddpalino** [comment]
Yes, this was already in progress (just took some time to get to it). I've opened #647 for this. It'll remain open for a couple days before I merge it, as it's a breaking change.

**Comment by macropin** [comment]
Thanks for fixing this. I came here to report this same issue and it looks like it's been fixed/merged. This issue can likely be closed.
